packages/lesswrong/server/callbacks/localgroupCallbacks.ts

```diff
@@ -14,3 +16,9 @@ getCollectionHooks("Localgroups").updateValidate.add((validationErrors: Array<an
   
   return validationErrors;
 });
+
+getCollectionHooks("Localgroups").updateAfter.add(async (newDoc: DbLocalgroup, {oldDocument}: {oldDocument: DbLocalgroup}) => {
```

Comment:
Q: Should this be updateAsync? We don't need to hold up the return of the new document on this.